### PR TITLE
add docker:publishLocal to requirements in readme

### PR DESCRIPTION
**Pull request by masonedmison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,23 +9,26 @@ Reference application developed in the [Functional event-driven architecture: Po
...
-   * [Tracing](#tracing)
-   * [Tests](#tests)
-   * [X Demo](#x-demo)
-* [Monitoring](#monitoring)
+- [trading](#trading)
+  - [Table of contents](#table-of-contents)
+  - [Web App](#web-app)
+  - [Overview](#overview)
+  - [Requirements](#requirements)
+    - [Running application](#running-application)
```
Comment (gvolpe) [inline README.md:17]: This addition is great but I would leave the rest as it was

**Review by gvolpe — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -69,6 +72,9 @@ Here's an overview of all the components of the system.
 
 The back-end application is structured as a mono-repo, and it requires both Apache Pulsar and Redis up and running. To make things easier, you can use the provided `docker-compose.yml` file.
 
+Note: The `docker-compose` file depends on declared services to be published on the local docker server. All docker builds are handled within the `build.sbt` using `sbt-native-packager`. To build all of the service images,
+run `sbt docker:publishLocal`.
```
[the remark this answers is not in the document]
Comment (masonedmison) [reply README.md]: Fixed :)

File: README.md (hunk 1 above)
Comment (masonedmison) [reply README.md]: Annoyingly, my markdown plugin made this change automatically. I switched it back.

File: README.md [hunk 3]
```diff
@@ -9,26 +9,26 @@ Reference application developed in the [Functional event-driven architecture: Po
...
-    - [Tracing](#tracing)
-    - [Tests](#tests)
-    - [X Demo](#x-demo)
-  - [Monitoring](#monitoring)
+* [trading](#trading)
+  * [Table of contents](#table-of-contents)
+  * [Web App](#web-app)
+  * [Overview](#overview)
+  * [Requirements](#requirements)
+    * [Running application](#running-application)
```
Comment (gvolpe) [inline README.md:17]: this should be the only line change in the ToC, the rest should remain unchanged :)

**Pull request closed without merging**